app/views/spree/shared/_braintree_hosted_fields.html.erb

```diff
@@ -25,4 +25,6 @@
   <div class="clear"></div>
   <input type="hidden" name="<%= prefix %>[payment_type]" value="<%= SolidusPaypalBraintree::Source::CREDIT_CARD %>">
   <input type="hidden" id="payment_method_nonce" name="<%= prefix %>[nonce]">
+  <% # hope we don't ever use this partial for real b/c this make noooooooooooo sense%>
+  <input type="hidden" id="braintree-device-data" name="order[payments_attributes][][braintree_device_data]" value="test123" />
```

Comment:
does this have to have _something_ in the value?  if it has to be there, i'd rather see something like an empty sting instead of a test string

Reply:
So I had been using that when trying to figure out how to get the data inserted from the request...... I guess i'll just remove it? I 'm not sure what else to do with it.

Reply:
This does look safe to remove since we don't use this partial anyway.